projects/storefrontstyles/scss/components/content/tab-paragraph-container/_tab-paragraph-container.scss

```diff
@@ -35,6 +35,10 @@ $useTabsOnPDP: false !default;
     &:hover {
       color: var(--cx-color-primary);
     }
+    &:focus {
+      z-index: 1;
+      transition: none;
```

Comment:
Would you mind moving this to the visible focus mixin as dicussed on slack?

Reply:
done